game/glutil.hh

```diff
@@ -41,7 +41,7 @@ namespace glutil {
 		
 		static GLsizeiptr size() { return sizeof(stereo3dParams); };
 		static GLintptr offset() { return alignOffset(shaderMatrices::size()); };
-		stereo3dParams() {};
+		stereo3dParams(): sepFactor(256), z0(260) {};
```

Comment:
Same

Reply:
> Hmmm what should the values of those variables be then? 0?

I think they're just not meant to have initial values.

Reply:
removed the initial values makes it work :)